tachyon/zk/lookup/halo2/prover_impl.h

```diff
@@ -223,17 +226,20 @@ void Prover<Poly, Evals>::Open(
   base::Ref<const Poly>(&polynomial.poly()), point##_ref, \
       polynomial.poly().Evaluate(point_set.point)
 
+  // NOTE(chokobole): The ORDER IS IMPORTANT!!!
```

Comment:
```suggestion
  // The ORDER IS IMPORTANT!!!
```
I thought we were removing NOTE(author) comments. https://github.com/kroma-network/tachyon/commit/98db4aa004b6202d9ca0660b74456f30f1dc2102